src/allotropy/parsers/appbio_quantstudio/appbio_quantstudio_structure.py

```diff
@@ -57,16 +58,17 @@ class Header:
 
     @staticmethod
     def create(reader: LinesReader) -> Header:
-        lines = [line.replace("*", "", 1) for line in reader.pop_until(r"^\[.+\]")]
+        lines = [line.strip() for line in reader.pop_until(r"^\[.+\]") if line.strip()]
+        if not lines:
+            msg = "Cannot parse data from empty header."
+            raise AllotropeConversionError(msg)
+
         csv_stream = StringIO("\n".join(lines))
-        raw_data = read_csv(csv_stream, header=None, sep="=", names=["index", "values"])
-        series = pd.Series(raw_data["values"].values, index=raw_data["index"]).astype(
-            str
+        raw_data = read_csv(
+            csv_stream, header=None, sep="=", skipinitialspace=True, index_col=0
         )
-        series.index = series.index.str.strip()
-        series = series.str.strip().replace("NA", None)
-
-        data = SeriesData(series)
+        raw_data.index = raw_data.index.str.replace("*", "")
+        data = df_to_series_data(raw_data.T.replace(np.nan, None))
```

Comment:
not quite following, why does this get transposed?

Reply:
df_to_series_data expects the data to be in a single row, when we read the data in directly, it's in a single column.

The previous code got around this by naming the columns "index" and "values" and manually creating a series.

Reply:
I think we do silly things like this all over the Lab Auto code too. You're the 🐐